hornetq-core-client/src/main/java/org/hornetq/core/client/impl/ClientConsumerImpl.java

```diff
@@ -132,7 +132,7 @@
 
    private boolean stopped = false;
 
-   private long forceDeliveryCount;
+   private volatile long forceDeliveryCount;
```

Comment:
I make this volatile because it can be accessed by a different thread (like in this case the connection failover thread calling getForceDeliveryCount()).